float32 rudder_demand
